internal/provider/datasources/account_member.go

```diff
@@ -158,10 +155,9 @@ func (d *AccountMemberDataSource) Read(ctx context.Context, req datasource.ReadR
 	// workspaceRoles, err := client.List(ctx, []string{model.Name.ValueString()})
 	accountMembers, err := client.List(ctx, []string{config.Email.ValueString()})
 	if err != nil {
-		resp.Diagnostics.AddError(
-			"Error refreshing Account Member state",
-			fmt.Sprintf("Could not search for Account Members, unexpected error: %s", err.Error()),
-		)
+		resp.Diagnostics.Append(helpers.ResourceClientErrorDiagnostic("Account Member", "list", err))
```

Comment:
not blocking, but wonder if we should make enums for the magical strings that we use for the resource type name

Reply:
I think you're right, crossed my mind while working on this. Added to https://github.com/PrefectHQ/terraform-provider-prefect/issues/203.